packages/styles/src/prepareStyle.ts

```diff
@@ -0,0 +1,32 @@
+import {
+    Style,
+    NativeStyle,
+    StyleUtils,
+    StyleObject,
+    NativeStyleUtils,
+    NativeStyleObject,
+} from './types';
+
+export const prepareStyle = <TProps extends object>(style: Style<TProps>) => style;
```

Comment:
I tried that but since object is used here soo many times in this file, it will make types like really long and harder to read, so I would prefer to keep it since it shoudn't affect type safety anyway.

Reply:
object is really abstract though :) I think modern libs also use Record for the "key: value" thing. Since as the ESLint docs point out - everything in JS in as object

Reply:
fixed